pkg/components/standalone_loader.go

```diff
@@ -40,18 +42,25 @@ func (s *StandaloneComponents) LoadComponents() ([]components_v1alpha1.Component
 		if !file.IsDir() {
```

Comment:
Can you please add new test for LoadComponents() as a simple go test ?

Reply:
+1 on new test.

Reply:
Sure. I will add.

Reply:
Added unit tests for decoding yamls.